Content.Server/Power/Components/ExtensionCableReceiverComponent.cs

```diff
@@ -18,5 +19,8 @@ public sealed partial class ExtensionCableReceiverComponent : Component
         [ViewVariables(VVAccess.ReadWrite)]
         [DataField("receptionRange")]
         public int ReceptionRange { get; set; } = 3;
+
+        [ViewVariables]
+        public GameTick ProviderConnectedTick { get; set; } = GameTick.Zero;
     }
```

Comment:
This looks like it's going to break mapsaving pretty hard because this never gets serialized.

Reply:
I didn't think about it. Hope changing the decorator to `[NonSerialized]` is enough to address the serialization issue.

Reply:
I removed it, and brought back [my previous solution](https://github.com/space-wizards/space-station-14/pull/23736). The fact that [anchoring events happen earlier than the startup events](https://github.com/space-wizards/RobustToolbox/blob/master/Robust.Shared/GameObjects/Systems/SharedTransformSystem.Component.cs#L321-L333) made me crazy. I even thought that connections are somehow serialized into the map files, before I realized that the anchoring event connects things before the ext. cable component startup.